tests/libvmi/factory.go

```diff
@@ -34,6 +34,18 @@ const (
 // NewFedora instantiates a new Fedora based VMI configuration,
 // building its extra properties based on the specified With* options.
 func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	return newFedora(cd.ContainerDiskFedora, opts...)
```

Comment:
This change appears in the first commit, but I think it does not belong here... right?
If it does, just update the commit message.

Reply:
Right, we need this ony for the SRIOV part I will add it to that commit.